web-client/src/presenter/presenter.ts

```diff
@@ -1197,6 +1202,7 @@ export const presenterSequences = {
     setSelectedAddressOnFormSequence as unknown as Function,
   setSelectedBatchIndexSequence:
     setSelectedBatchIndexSequence as unknown as Function,
+  setSelectedDocumentsForDownloadSequence,
```

Comment:
Why don't some of these have `as unknown as Function`?

Reply:
`setSelectedDocumentsForDownloadSequence` is typed already in its declaration. `web-client/src/presenter/sequences/setSelectedDocumentsForDownloadSequence.ts`